src/Model/DatagridManagerInterface.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\AdminBundle\Model;
+
+/**
+ * A datagrid manager is a bridge between the model classes and the admin datagrid functionality.
+ *
+ * @method array getDefaultPerPageOptions(string $class)
+ */
+interface DatagridManagerInterface
+{
+    /**
+     * Return _sort_order, _sort_by, _page and _per_page values.
+     *
+     * @param string $class
+     *
+     * @return array<string, int|string>
+     */
+    public function getDefaultSortValues($class);
```

Comment:
```suggestion
    public function getDefaultSortValues($class): array;
```

Reply:
I can't. This would be a BC-break.

Reply:
Oh right I did not notice the inheritance